### DISCUSSION TITLE
Import Different stylesheet based on layout

### DISCUSSION THREAD

**Discussion by dbzx10299** [nuxt/nuxt#19709] [Questions] [answered]
Is there a way to import a css stylesheet into a layout. I have a block of code like this in the `default.vue` layout
```js
<script setup>
useHead({
  link: [
    {
       rel: 'stylesheet',
       href: '@/assets/main.css' 
    } 
  ]
})
</script>

<template>
  <slot />
</template>
```

But it keeps throwing an error

**Comment by Najidnadri** [comment, marked as answer]
The problem with this method is webpack cannot read the through the `useHead` function. 

One way you can do this is by adding your custom CSS files inside the `public` directory and call it inside the `useHead`.

Your project folder will be like this -
```
-package.json
-README.md
-nuxt.config.ts
-layouts
-public
  -defaultlayout.css
  -layout1.css
```

`default.vue` layout 
```
<script setup>
useHead({
  link: [
    {
       rel: 'stylesheet',
       href: './defaultlayout.css' 
    } 
  ]
})
</script>
```

`layout.vue` layout
```
<script setup>
useHead({
  link: [
    {
       rel: 'stylesheet',
       href: './layout1.css' 
    } 
  ]
})
</script>
```

**Reply by dbzx10299** [reply]
So in your example the css will not be minified correct?

**Reply by Najidnadri** [reply]
Yes unfortunately
